ci/tests/general/general_alba.py

```diff
@@ -635,8 +635,9 @@ def filter_disks(disk_names, amount, disk_type):
         filtered_disks = dict()
         for node_id in disk_names.iterkeys():
             node_ids.append(node_id)
+        no_of_nodes = len(node_ids)
         while count < amount:
-            node_id = node_ids[count % 3]
+            node_id = node_ids[count % no_of_nodes]
```

Comment:
I'd use len(node_ids) ... no need for an extra variable here
if still necessary use nr_of_nodes io no_of_nodes